### DISCUSSION TITLE
怎么使用代码块分组？

### DISCUSSION THREAD

**Discussion by leoleor** [vuepress-theme-hope/vuepress-theme-hope#4053] [Q&A] [answered]
"vuepress": "2.0.0-beta.62",
"vuepress-theme-hope": "2.0.0-beta.222"

https://vuepress-theme-hope.github.io/v1/zh/guide/markdown/code-group.html
按照文档这里的步骤如下：
```js
module.exports = {
  themeConfig: {
    mdEnhance: {
      codegroup: true,
    },
  },`
};
```

:::: code-group

::: code-group-item yarn

```bash
yarn add -D vuepress-theme-hope
```

:::

::: code-group-item npm:active

```bash
npm i -D vuepress-theme-hope
```

:::

::::

但是实际codegroup属性会报错：对象字面量只能指定已知属性，并且“codegroup”不在类型“Omit<MarkdownEnhanceOptions, "container"> & { container?: boolean | undefined; }”中。

这是怎么回事？

**Comment by Mister-Hope** [comment, marked as answer]
v1已经停止维护很多年了。

**Reply by leoleor** [reply]
啊哈，尴尬了，我找到了，谢谢
